DesignSafe-CI/portal
# Task/DES-2030, 2104, 2105 - Publication Header UI Changes

## Overview: ##
This PR includes design changes requested by Ellen, Craig, and Maria.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2030](https://jira.tacc.utexas.edu/browse/DES-2030)
* [DES-2104](https://jira.tacc.utexas.edu/browse/DES-2104)
* [DES-2105](https://jira.tacc.utexas.edu/browse/DES-2105)

## Summary of Changes: ##
1. Relocated "View Data Diagram", "View Data Metrics",  "Leave Feedback" above the "Description" field
2. Added icon for feedback link
3. Created Show More component to show more/less of a publication's description depending on how long it is
4. Changed "Download Dataset" to a standard button

## Testing Steps: ##
1. Look at published projects for Other, Experimental, Simulation, Hybrid Simulation, and Field Research
2. Check for the changes in the images included below
3. The "Show More/Less" should only appear if the description is more than 4 lines long. It should always start cropped down to 4 lines with the "Show More" button visible below

## UI Photos:
![less](https://user-images.githubusercontent.com/29575979/134740458-06880ce1-cc54-4039-9547-076f1db5bf9e.png)
![more](https://user-images.githubusercontent.com/29575979/134740452-d31b63bb-7342-4189-beed-2070835951ad.png)
